src/frontend/src/utils/work-package.utils.ts

```diff
@@ -11,3 +11,10 @@ export const getTitleFromFormType = (formType: WPFormType, wbsElement: WbsElemen
       return `${wbsPipe(wbsElement.wbsNum)} - ${wbsElement.name}`;
   }
 };
+
+export const getUpcomingWorkPackages = (workPackages: WorkPackage[]): WorkPackage[] => {
+  const currentTime = new Date();
+  const twoWeeks = new Date();
+  twoWeeks.setDate(currentTime.getDate() + 14);
+  return workPackages.filter(({ startDate }) => currentTime <= startDate && startDate <= twoWeeks);
```

Comment:
Just make it so that work packages with a start date of the current date are filtered out